### ISSUE TITLE
profile avatar images are not shown in `PageDelegateVote`

### ISSUE THREAD

**Issue by fisx** [liqd/aula#992]
may be a deployment issue?

**Comment by fisx** [comment]
I'll look into this.

**Comment by fisx** [comment]
the img tags point to `/static/avatars`, so as a quick work-around i did this:

```shell
$ cd deployments/gymnasium-nottuln.aula.de/static
$ ln -s ../avatars/
$ cd ../../jenaplan-schule-jena.aula.de/static
$ ln -s ../avatars/
$ cd ../../pestalozzi-rs-fr.aula.de/static
$ ln -s ../avatars/
$ cd ../../schule-am-hafen.aula.de/static
$ ln -s ../avatars/
```

(same for `demo1.aula.de`.)

**Comment by andorp** [comment]
Is it a configurational issue?

**Comment by fisx** [comment]
Fixed in #995.  See also: #996.
